ms.html

```diff
@@ -129,6 +131,10 @@
                     externalData.innerHTML = data
                 })
         }
+
+        async function addRecordToManagedCollection(){
```

Comment:
You put the rest of these in the template logic. Is this waiting here on purpose?

Reply:
Ah crap it's just left over code junk.  This should be deleted.